pasqal_cloud/authentication.py

```diff
@@ -44,6 +44,7 @@ class ExpiringTokenProvider(TokenProvider, ABC):
     this class and defining your own `_query_token` method. See Auth0TokenProvider for
     an implementation example.
     """
+
```

Comment:
should we add a linting stage to the project to check that things have been formatted correctly?

In other PASQAL project we use `pre-commit` hooks a lot, checking black, and flake8 at the very least.

Reply:
I added a black check in the github actions. The `pre-commit` hook can be added in another MR, I created the following issue https://github.com/pasqal-io/pasqal-cloud/issues/82

Reply:
thanks!